lib/YaST/Firstboot/WelcomeClient.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.    
+
+package YaST::Firstboot::WelcomeClient;
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init(); 
+}
+
+sub init {
+    my ($self, $debug_label) = @_;
+    $self->{btn_next}            = $self->{app}->button({id => 'next'});
+    $self->{client_debug_label} = $self->{app}->debug_label({'debug_label' => 'Not Welcome'});
```

Comment:
This is another culprit, url requests do not support `debug_label` parameter, it just uses `label` to match both properties. I might be wrong on this one, but I would suggest trying `label property.

Reply:
I tried that, the label fails and debug_label works eg http://server:port/v1/widgets?debug_label=%22Language%20and%20Keyboard%20Layout%22 fails if we replace debug_label by label. SO I tried to create a new wiget here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/71f910fee2711409c687e7d3b02b000d2fe87d50/lib/YuiRestClient/Widget/DebugLabel.pm just copied from label

Reply:
You anyway do not need a new widget for it, because it's search filter, so you can have `$self->{app}->label({'debug_label' => 'Not Welcome'});`